### PR TITLE
Github Abmonition Feature

### PR DESCRIPTION
**Pull request by mihaiandreineacsu**

### CODE REVIEW

**Review by spmse — COMMENTED** [review]
Nice one! 

In the best case add some more documentation text into the mdx file, so that people know about the component and how to use it.

After documentation is complete we can merge this branch

File: src/components/GithubLinkAdmonition/index.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+type GithubLinkAdmonitionType = {
+  link: string;
+  text?: string;
+  type?: string;
+  title?: string;
+  children?: ReactNode
+};
+
```
Comment (henrymanke) [inline src/components/GithubLinkAdmonition/index.tsx:13]:
```suggestion
type AdmonitionType = "info" | "tip" | "warning" | "note" | "danger";
```

File: src/components/GithubLinkAdmonition/index.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React, {ReactNode} from 'react';
+import Admonition from '@theme/Admonition';
+import GithubIcon from '/static/img/github.svg';
+
+
+type GithubLinkAdmonitionType = {
+  link: string;
+  text?: string;
+  type?: string;
```
Comment (henrymanke) [inline src/components/GithubLinkAdmonition/index.tsx:9]:
```suggestion
  type?: AdmonitionType;
```

**Review by henrymanke — CHANGES_REQUESTED** [review]
Hi @mihaiandreineacsu,
we add some minor improvements.

**Pull request closed without merging**